geography/README.md

```diff
@@ -0,0 +1,254 @@
+
+The publishing Agency should establish and communicate to providers how frequently these files should be polled.
+
+The `updated` field in the payload wrapper should be set to the time of publishing a revision, so that it is simple to identify a changed file.
+
+[Top][toc]
+
+### Response Format
+
+See the [Responses][responses] and [Error Messages][error-messages] sections.
+
+[Top][toc]
+
+### Authorization
+
+When making requests, the Geography API expects `provider_id` to include an `access_token` in the `Authorization` header, in the form `Authorization: Bearer <access_token>`. The token issuance, expiration and revocation policies are at the discretion of the Agency.
+
+Optionally, an Agency may decide to make these endpoints unauthenticated and public, which could be done in conjunction with the [/policy](/policy) endpoints.
+
+[Top][toc]
+
+## Schema
+
+Placeholder -- link to schema to be added later.  
+
+[Top][toc]
+
+### Geography Fields
+
+| Name               | Type      | Required/Optional | Description                                                                     |
+| ----------------   | --------- | --- | --------------------------------------------------------------------------------------------- |
+| `name`             | String    | Required   | Name of geography                                                                      |
+| `description`      | String    | Optional   | Detailed description of geography                                                      |
+| `geography_type`   | Enum      | Optional   | Type of geography, e.g. `municipal_boundary` or `council_district`.  See [Geography Types](#geography-types). |
+| `geography_id`     | UUID      | Required   | Unique ID of geography                                                                 |
+| `geography_json`   | UUID      | Required   | The GeoJSON that defines the geographical coordinates.                                 |
+| `effective_date`   | [timestamp][ts] | Optional   | The date at which a Geography is considered "live".  Must be at or after `publish_date`. |
+| `publish_date`     | [timestamp][ts] | Required   | Time that the geography was published, i.e. made immutable                       |
+| `prev_geographies` | UUID[]    | Optional   | Unique IDs of prior geographies replaced by this one                                   |
+
+[Top][toc]
+
+### Previous Geographies
+
+Obsoleting or otherwise changing a geography is accomplished by publishing a new geography with the `prev_geographies` field, which is a list of UUID references to the geography or geographies superseded by the new geography. The previous geographies are also published in the `/geographies` endpoint.  Using it allows agencies to look back historically at previously published geographies, for analysis, historic reference, or an auditable change trail.
+
+This field is optional can be omitted by the publishing Agency.  
+
+### Geography Type
```

Comment:
No specific suggestions here, but some of these Geography types feel like Policies more than descriptions of physical space. Do we want to mix those concerns?

Reply:
Do you mean for Previous Geographies, or Geography Type?  For the types I think they can align and overlap with policies, but not always (eg census).  Many of them are in the examples as geographies that cities define, so we want to call them out and name them here.

Reply:
For the Types. Trying to think from an implementation perspective, how does a geography with type `no_ride_zone` or `slow_ride_zone` differ from the equivalent Policy document? If an agency publishes a Geography without a corresponding Policy, are companies expected to follow the type of the Geography as an implicit "rule"?

Reply:
Ah I could see the confusion. Maybe we need some language to clarify that the types do not equal policy, but instead are for organizational and discovery purposes in the Geography API.  They need to be linked to policy or other areas of MDS to be meaningfully used.

Reply:
Yeah that could work. Another idea is to collapse those down to `policy_zone` or similar, to make it explicit that an associated Policy is needed. Then leave the details to the Policy/Rules. Would also make reusing the Geography across multiple (perhaps totally different) Policies more natural.

Reply:
"Geography types do not equal policy, but instead are for organizational and discovery purposes within the standalone Geography API. They need to be linked to policy or other areas of MDS to be meaningfully used."

Reply:
But that isn't necessarily the case for all Geography types. The Agency may wish to publish their official boundary under `/geographies`, not to tie it to Policy but to have a consistent and known publishing mechanism. I think part of the reason this API was moved out from Policy is to not tie it so explicitly to Policy, right?

Reply:
I think clarifying language like this would work. Geographies could be used in more areas than just policy, and I think that geographies defined for some initial purpose align with how cities work, even if those areas can be used for multiple different things later.